main 2.py

```diff
@@ -0,0 +1,130 @@
+def mainScreen():
+    print("1. Save a New Entry")
+    print("2. Search By Id")
+    print("3. Print Ages Average")
+    print("4. Print All Names")
+    print("5. Print All IDs")
+    print("6. Print All Entries")
+    print("7. Print Entry By index")
+    print("8. Exit")
+
+
+def checkIfNumber(user_input):
+    char_flag = True
+    for num in user_input:
+        if num != "9" and num != "8" and num != "7" and num != "6" and num != "5" and num != "4" and num != "3" and num != "2" and num != "1" and num != "0":
+           char_flag = False
+    return char_flag
+    
+    
+def userChoseone(my_dic):
+        while True:
+            user_id = input("Enter Id Number ")
+            user_age = input("Enter User Age ")
+            if checkIfNumber(user_id) and checkIfNumber(user_age) == True:
+                user_id = int(user_id)
+                user_age = int(user_age)
+                break
+            else:
+                print("Something Wrong, Id and Age Should Be a Numbers Try Again ")
+        user_name = input("Enter User Name ")
+        my_dic[user_id] = [user_name, user_age]  
+        print("ID [" +str(user_id)+ "] Added Succesfully")
+        input("Press Enter To Continue")
+        return my_dic
+        
+def userChoseTwo(user_dict):
+    id_search = int(input("Please What is The Id to look For "))
+    if id_search in user_dict:  
+        print("ID : " +str(id_search))
+        print("Name : " +user_dict[id_search][0])
+        print("Age : " +str(user_dict[id_search][1]))
+        input("Press Enter to Continue")
+    else:
+        input("User Not Found, Press Enter To Back to Main")
+        
+
+def userChoseThree(user_dict):
+    sum = 0
+    for age in user_dict:
+        sum += user_dict[age][1]
+    avg = sum / len(user_dict)    
+    return avg
+    
+def userChoseFour(user_dict):
+    print("The Name Are : ")
+    for names in user_dict:
+        print(user_dict[names][0])
+    input("Press Enter to Continue")
```

Comment:
השורה input("Press Enter to Continue") לא יכולה להופיע הרבה פעמים בקוד
היא צריכה להופיע **רק פעם אחת**
בסוף של הפונקציה main